src/styles/main.scss

```diff
@@ -1,3 +1,45 @@
+$cell-size: 300px;
+
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($cell-size, 1fr));
+  grid-template-rows: repeat(6, $cell-size);
```

Comment:
Because of this line you constantly have 6 rows. When you stretch the window beyond 600px, those rows are still there which results into empty white space. Not necessarily a mistake, but something to be aware of.